recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -24,9 +24,30 @@ ANDROID_VARIANT ?= "eng"
 SOC_FAMILY ?= "default"
 
 ANDROID_OUT_DIR_COMMON_BASE = ""
-DDK_KM_PREBUILT_MODULE ?= ""
-TARGET_PREBUILT_KERNEL ?= ""
-DDK_UM_PREBUILDS ?= ""
+
+python __anonymous () {
+    def create_param_str():
```

Comment:
I think, this function can be named better. It not just creates parameter, it checks for a path. So, something like `process_prebuilt_path` would be much better.

Reply:
there is no any action with the prebuild path.